openApiDocs/beta/Bookings.yml

```diff
@@ -632,7 +632,7 @@ paths:
       tags:
         - bookingBusinesses.Actions
       summary: Invoke action cancel
-      operationId: bookingBusinesses.appointments_cancel
+      operationId: bookingBusinesses.bookingBusiness.appointments.bookingAppointment_cancel
```

Comment:
This change in operationId will require us to update several directives and may also throw off AutoREST. I'm waiting for the build to complete the assess the extent of the change. @irvinesunday, can we turn this off when `style=Powershell` is set for now.

Reply:
For instance, AutoREST generates the following command name from this operationId:
| Before | After |
|-|-|
|`Stop-MgBookingBusinessAppointment` | `Stop-MgBookingBusinessAppointmentBookingAppointment`|

Reply:
@peombwa so we wouldn't want to consider changing these OperationIds ever, even in the future?